### ISSUE TITLE
(OUI Docs) Icons

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/oui#248]
Section: Logos
**NOTE:** Remove indicates ONLY removing from documentation. Do NOT remove these logos from the icon set within the repo. These logos will be deprecated from the actual icon library in another issue, once investigation is done to see which ones still remain in use.
- [x] Change header to OpenSearch logos
- [x] Remove logoAppSearch
- [x] Remove logoBeats
- [x] Remove logoBusinessAnalytics
- [x] Remove logoCloud
- [x] Remove logoCloudEnterprise
- [x] Remove logoCode
- [x] Remove logoElastic
- [x] Remove logoElasticStack
- [x] Remove logoElasticsearch
- [x] Remove logoEnterpriseSearch
- [x] Remove logoKibana
- [x] Remove logoLogging
- [x] Remove logoLogstash
- [x] Remove logoMaps
- [x] Remove logoMetrics
- [x] Remove logoObservability
- [x] Remove logoSecurity
- [x] Remove logoSiteSearch
- [x] Remove logoUptime
- [x] Remove logoWorkplaceSearch

- [x] Add logoOpenSearch with https://playground.opensearch.org/ui/default_branding/opensearch_mark_default_mode.svg 

Section: Apps
Note: 
- Remove only means to remove them from documentation. Because many of these are in use as glyphs, there will need to be investigation of how many are still in use in the application, generalized and then campaigns will need to be done to implement the Glyph version vs the App Icon version. That work is tracked separately [here](https://github.com/opensearch-project/oui/issues/249)

- [x] Remove addDataApp

**Comment by KrooshalUX** [comment]
@kgcreative are you able to upload the svg for the OpenSearch logo?

**Comment by KrooshalUX** [comment]
blocked by #369

**Comment by ananzh** [comment]
add one more task:
https://github.com/opensearch-project/oui/issues/255 (3rd task)
